submissions/eoneoff/data-structures-http/server/handler.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable default-case */
+/* eslint-disable no-return-assign */

```

Comment:
Where was this error shown?

Reply:
Default case for switch statement without default (in handling list POST and DELETE).  no-return-assign for line 117 ( request.on('data', chunk => body += chunk.toString());). Eslint wants arrow functions not to return assignments (what if I have put = instead of ==). If you make a single-statement arrow function, a return keyword is implied, so it is something like return body+=chunk.toString(). It can be bypassed by making it  request.on('data', chunk => {body += chunk.toString()});